lib/SyTest/HomeserverFactory/Synapse.pm

```diff
@@ -102,6 +102,15 @@ sub create_server
    return $self->{impl}->new( %params );
 }
 
+sub get_critical_tests
+{
+   return (
+       'POST [^ ]+ register can create a user',
```

Comment:
it's still more idiomatic to do \S (meaning "not whitespace"), but not a big deal.